app/views/practices/implementation_forms/_resources.html.erb

```diff
@@ -0,0 +1,130 @@
+<% provide :head_tags do %>
+  <%= javascript_include_tag '_practice_editor_utilities', 'data-turbolinks-track': 'reload' %>
+<% end %>
+<%
+  resource_type = resource_type
+  resource_type_label = resource_type_label
+  media_type = media_type
+  slug = resource_type + "-" + resource_type_label + "-" + media_type
+  link_to_add_link_slug = 'link_to_add_link_' + resource_type + "_" + resource_type_label + "_" + media_type
+  link_to_add_button_slug = 'link_to_add_button_' + resource_type + "_" + resource_type_label + "_" + media_type
+  f = f
+  add_practice_resources_link = "add-practice-resources-link-" + slug
+  #practice_resources_fields_blueprint
+  blue_print_id = "practice_resources_fields_blueprint_" + slug
+  resource_type_place_holder = "RESOURCE_TYPE_PLACE_HOLDER"
+  resource_type_label_place_holder = "RESOURCE_TYPE_LABEL_PLACE_HOLDER"
+  media_type_place_holder = "MEDIA_TYPE_PLACE_HOLDER"
+
+  li_class_place_holder = "LI_CLASS_PLACE_HOLDER"
+%>
+
+<div id="<%= slug %>-resources" class="desktop:grid-col-10 grid-col-12 p0">
+  <section class="usa-section padding-top-0 p0-bottom">
+    <fieldset class="usa-fieldset">
+      <legend class="usa-sr-only">Core resources</legend>
+      <div id="<%= slug %>-container"
+           class="margin-top-1">
+        <h3 class="text-normal font-sans-sm line-height-sans-5 margin-top-1">
+          <% if resource_type_label === "people" %>
+            <b>People: </b>Type a job title, department, and/or discipline another facility would need to involve in implementing your practice.  Provide dependencies for implementation (e.g., Clinical Application Controller required for 2-4 hours/week for 1-2 weeks).
```

Comment:
any of these `<b>` tags should be `span` tags using the uswds class `text-bold`

Reply:
Updated with span tags